content/hi/cluster.md

```diff
@@ -0,0 +1,19 @@
+---
+title: क्लस्टर (Cluster)
+status: Completed
+category: अवधारणा
+tags: ["infrastructure", "architecture", ""]
+---
+
+## यह क्या है
+
+क्लस्टर कंप्यूटर या एप्लिकेशन का एक समूह है जो एक सामान्य लक्ष्य की दिशा में मिलकर काम करता है। क्लाउड नेटिव कंप्यूटिंग के संदर्भ में, यह शब्द अक्सर [कुबेरनेट्स](/kubernetes/) पर लागू होता है। कुबेरनेट्स क्लस्टर सेवाओं (या वर्कलोड) का एक समूह है जो अपने स्वयं के कंटेनरों में चलता है, आमतौर पर विभिन्न मशीनों पर। एक नेटवर्क से जुड़ी इन सभी [कंटेनरीकृत](/containerization/) सेवाओं का संग्रह एक क्लस्टर का प्रतिनिधित्व करता है।
+
+## समस्या
+
+एकल कंप्यूटर पर चलने वाला सॉफ़्टवेयर विफलता का एकल बिंदु प्रस्तुत करता है - यदि वह कंप्यूटर क्रैश हो जाता है, या कोई गलती से पावर केबल को अनप्लग कर देता है, तो कुछ व्यवसाय-महत्वपूर्ण प्रणाली ऑफ़लाइन हो सकता है। इसलिए आधुनिक सॉफ़्टवेयर को आम तौर पर [वितरित एप्लिकेशन](/distributed-apps/) के रूप में बनाया जाता है, जिसे क्लस्टर के रूप में एक साथ समूहीकृत किया जाता है।
+
+## समाधान
+
+क्लोस्ट्रेड, वितरित ऍप्लिकेशन्स कई मशीनों में चलते हैं, विफलता के एक बिंदु को समाप्त करते हैं। लेकिन वितरित प्रणालियों का निर्माण वास्तव में कठिन है। वास्तव में, यह अपने आप में एक कंप्यूटर विज्ञान अनुशासन है। श्विक प्रणालियों की आवश्यकता और वर्षों के परीक्षण और त्रुटि के कारण एक नए प्रकार के तकनीकी स्टैक का विकास हुआ:
```

Comment:
1. the word clustered can be written in a better way than `क्लोस्ट्रेड` so that the pronunciation matches. in fact we should look for a word that means clustered in Hindi.
2. would recommend finding replacement words for `श्विक` and `त्रुटि` as these are not very commonplace and may add to overhead instead of adding ease

rest looks good!